### DISCUSSION TITLE
Czarna lista stron/domen generowanych przez AI

### DISCUSSION THREAD

**Discussion by rjolina** [MajkiIT/polish-ads-filter#22433] [Ideas]
Mówię serio - **zapłaciłbym** za jakąś listę do ublocka, która będzie blokowała/wycinała z wyników wyszukiwania strony z artykułami generowanymi przez AI. Wuj jasny strzela jak widzę januszy biznesu, którzy psują internet swoimi "projektami" :/

![Przechwycenie obrazu ekranu_2024-05-21_17-37-58](https://github.com/MajkiIT/polish-ads-filter/assets/83005066/e9eb17b8-eef4-4194-b7f0-6504e91f00f5)

**Comment by blocker999** [comment]
A KAD znasz ?

https://github.com/FiltersHeroes/KAD

**Reply by krystian3w** [reply]
Od kiedy on blokuje bełkot AI, nie kojarzę by ktoś to zgłaszał do zaadresowania ani by było to w kompetencjach CERT (nie każdy bełkot kradnie dane np. karty kredytowej) lub Legalnie w Sieci (nie każdy bełkot to fejkowy sklep), te jego 4 tysiące notatek blogowych raczej nie przypomina sklepu ani formularzy phishingowych.

**Comment by krystian3w** [comment]
Niby komuś się chciało:

https://github.com/alvi-se/ai-ublock-blacklist

> You can [click here](https://subscribe.adblockplus.org/?location=https%3A%2F%2Fraw%2Egithubusercontent%2Ecom%2Falvi%2Dse%2Fai%2Dublock%2Dblacklist%2Fmaster%2Flist%2Etxt&title=AI%20Content%20Farms) to subscribe to this list automatically. This link works only if you have uBlock Origin installed.

Jedynie nie wiem na ile blokuje polskie biznesy tego pana ze zdjęcia.

Manualne łapanie i bazowanie na społeczności raczej temu zaprzecza.
